scale-encode/src/impls/mod.rs

```diff
@@ -701,7 +712,7 @@ mod test {
         // These don't impl TypeInfo so we have to provide the target type to encode to & compare with:
         assert_value_roundtrips_to(Arc::new("hi"), "hi".to_string());
         assert_value_roundtrips_to(Rc::new("hi"), "hi".to_string());
-        // encodes_like_codec(std::time::Duration::from_millis(123456));
+        // encodes_like_codec(core::time::Duration::from_millis(123456));
```

Comment:
We should really "uncomment this" or remove it completely.

Probably something missed when support for `core::time::Duration` was added.